snapcraft.yaml

```diff
@@ -0,0 +1,58 @@
+name: google-play-music-desktop-player
+version: '4.1.1'
+summary: Standalone player for Google Play Music
+description: |
+  This Google Play Music player provides desktop integration - with music
+  hotkeys and the sound indicator via an MPRIS interface, the abiliy to use
+  custom colours and load custom themes, a remote-control interface, and
+  lower resource consumption than a full-blown browser tab.
+  .
+  This is *not* an official Google project.
+grade: stable
+confinement: strict
+icon: snap/gui/google-play-music-desktop-player.png
```

Comment:
Sadly, no. The `snapcraft` tool checks for the presence of the `icon:` file at the beginning of the build.

Oh. But it accepts SVG, so it *could* point to `src/assets/icons/svg/vector_logo.svg`. Is there any reason not to do that?